src/openapi.yaml

```diff
@@ -324,6 +318,98 @@ paths:
       - JWT-Claims:
         - documenten.aanmaken
     parameters: []
+  /enkelvoudiginformatieobjecten/_zoek:
+    post:
+      operationId: enkelvoudiginformatieobject__zoek
+      description: Opvragen en bewerken van (ENKELVOUDIG) INFORMATIEOBJECTen (documenten).
```

Comment:
```suggestion
      description: Opvragen van (ENKELVOUDIG) INFORMATIEOBJECTen (documenten) op basis van zoekcriteria.
```

Reply:
Het is immers geen bewerkactie maar een query (met zoek parameters in de request body)

Reply:
summary: Voer een zoekopdracht uit op (ENKELVOUDIG) INFORMATIEOBJECTen .
      description: "Zoeken/filteren gaat normaal via de `list` operatie, deze is echter
        niet geschikt voor zoekopdrachten met UUIDs."